lib/Model/Attendee.php

```diff
@@ -99,74 +99,29 @@ class Attendee extends Entity {
 	public const PERMISSIONS_MODIFY_REMOVE = 'remove';
 	public const PERMISSIONS_MODIFY_ADD = 'add';
 
-	/** @var int */
-	protected $roomId;
-
-	/** @var string */
-	protected $actorType;
-
-	/** @var string */
-	protected $actorId;
-
-	/** @var null|string */
-	protected $displayName;
-
-	/** @var null|string */
-	protected $pin;
-
-	/** @var int */
-	protected $participantType;
-
-	/** @var bool */
-	protected $favorite;
-
-	/** @var int */
-	protected $notificationLevel;
-
-	/** @var int */
-	protected $notificationCalls;
-
-	/** @var int */
-	protected $lastJoinedCall;
-
-	/** @var int */
-	protected $lastReadMessage;
-
-	/** @var int */
-	protected $lastMentionMessage;
-
-	/** @var int */
-	protected $lastMentionDirect;
-
-	/** @var int */
-	protected $readPrivacy;
-
-	/** @var int */
-	protected $permissions;
-
-	/** @var string */
-	protected $accessToken;
-
-	/** @var string */
-	protected $remoteId;
-
-	/** @var string */
-	protected $invitedCloudId;
-
-	/** @var null|string */
-	protected $phoneNumber;
-
-	/** @var null|string */
-	protected $callId;
-
-	/** @var int */
-	protected $state;
-
-	/** @var int */
-	protected $unreadMessages;
-
-	/** @var int */
-	protected $lastAttendeeActivity;
+	protected int $roomId = 0;
+	protected string $actorType = '';
+	protected string $actorId = '';
+	protected ?string $displayName = null;
+	protected ?string $pin = null;
+	protected int $participantType = 0;
+	protected bool $favorite = false;
+	protected int $notificationLevel = 0;
+	protected int $notificationCalls = 0;
+	protected int $lastJoinedCall = 0;
+	protected int $lastReadMessage = 0;
+	protected int $lastMentionMessage = 0;
+	protected int $lastMentionDirect = 0;
+	protected int $readPrivacy = 0;
+	protected int $permissions = 0;
+	protected string $accessToken = '';
+	protected string $remoteId = '';
+	protected string $invitedCloudId = '';
+	protected ?string $phoneNumber = null;
+	protected ?string $callId = null;
+	protected int $state = 0;
+	protected int $unreadMessages = 0;
```

Comment:
Tested with this line only - works

The rest is breaking most of the things for now, even POST participants returns 500. Should also be the reason for integration tests to fail?